department-of-veterans-affairs/abd-vro
# dev_* image prefixing removal

## What was the problem?

No longer needing a distinction between dev_* and non prefixed images

Associated tickets or Slack threads:
- #2759 
- Obsoletes #2470 

## How does this fix it?[^1]

* modifies/removes workflows to avoid publishing dev_* images and references to those

## How to test this PR
- Successful run of the SecRel workflow: https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/runs/8380517089


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
